src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -118,6 +121,15 @@ bool ov::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ngraph::F
     // before CommonOptimization pipeline execution
     manager.register_pass<ngraph::pass::MOCTransformations>(true, false);
 
+    // Marks constant folded shape values to be kept in FP32.
+    // When dynamism on GPU will be turned on
+    // need to keep the whole shape subgraph not only constants
+    manager.register_pass<ov::pass::MarkPrecisionSensitiveSubgraphs>();
+    // Be default were disabled. Only plugins that support mixed inference in FP16/FP32
```

Comment:
```suggestion
    // By default were disabled. Only plugins that support mixed inference in FP16/FP32
```

Reply:
in the version comment is not present anymore